test/unit_tests/apps/owning-a-home/js/explore-rates/dom-values-spec.js

```diff
@@ -0,0 +1,43 @@
+const BASE_JS_PATH = '../../../../../../cfgov/unprocessed/apps/owning-a-home/';
+const chai = require( 'chai' );
+const expect = chai.expect;
+const sinon = require( 'sinon' );
+let sandbox;
+
+const HTML_SNIPPET =
+  `<input id="credit-score"
+          type="range" min="600" max="840" step="20" value="700">
+  <select id="arm-type">
+      <option value="3-1">3/1</option>
+      <option value="5-1">5/1</option>
+      <option value="7-1">7/1</option>
+      <option value="10-1">10/1</option>
+  </select>
+  <input id="test-price" type="text" value="$300,000">
+  <input id="house-price" type="text" placeholder="200,000">`;
+
+const domValues = require( BASE_JS_PATH + 'js/explore-rates/dom-values' );
+
+describe( 'explore-rates/dom-values', () => {
+  before( () => {
+    this.jsdom = require( 'jsdom-global' )( HTML_SNIPPET );
+  } );
+
+  after( () => this.jsdom() );
+
+  beforeEach( () => {
+    sandbox = sinon.sandbox.create();
+    document.body.innerHTML = HTML_SNIPPET;
+  } );
+
+  afterEach( () => {
+    sandbox.restore();
+  } );
+
+  it( 'should be able to get a value', () => {
```

Comment:
This is definitely better than no tests, but is more like a browser test than a unit test. It also doesn't account for the condition where it exists early with `undefined`

Reply:
Sorry, where what exists early?

Reply:
This one that was moved to the other PR. https://github.com/cfpb/cfgov-refresh/pull/3830#discussion-diff-171411382R14

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3830/commits/7e1c421a579f3b77bd8bb593c60747a17fa53cc1